spring-native-configuration/src/main/java/org/springframework/integration/IntegrationHints.java

```diff
@@ -0,0 +1,123 @@
+						org.springframework.integration.http.config.HttpContextUtils.class,
+						org.springframework.integration.websocket.config.WebSocketIntegrationConfigurationInitializer.class
+				}),
+		resources = @ResourceHint(patterns = "META-INF/spring.integration.properties"),
+		types = {
+				@TypeHint(access = AccessBits.FULL_REFLECTION,
+						types = {
+								org.springframework.integration.gateway.RequestReplyExchanger.class,
+								org.springframework.integration.graph.Graph.class,
+								org.springframework.integration.graph.CompositeMessageHandlerNode.class,
+								org.springframework.integration.graph.DiscardingMessageHandlerNode.class,
+								org.springframework.integration.graph.EndpointNode.class,
+								org.springframework.integration.graph.ErrorCapableCompositeMessageHandlerNode.class,
+								org.springframework.integration.graph.ErrorCapableDiscardingMessageHandlerNode.class,
+								org.springframework.integration.graph.ErrorCapableEndpointNode.class,
+								org.springframework.integration.graph.ErrorCapableMessageHandlerNode.class,
+								org.springframework.integration.graph.ErrorCapableRoutingNode.class,
+								org.springframework.integration.graph.IntegrationNode.class,
+								org.springframework.integration.graph.LinkNode.class,
+								org.springframework.integration.graph.MessageChannelNode.class,
+								org.springframework.integration.graph.MessageGatewayNode.class,
+								org.springframework.integration.graph.MessageHandlerNode.class,
+								org.springframework.integration.graph.MessageProducerNode.class,
+								org.springframework.integration.graph.MessageSourceNode.class,
+								org.springframework.integration.graph.PollableChannelNode.class,
+								org.springframework.integration.graph.ReceiveCounters.class,
+								org.springframework.integration.graph.RoutingMessageHandlerNode.class,
+								org.springframework.integration.graph.SendTimers.class,
+								org.springframework.integration.graph.TimerStats.class,
+								org.springframework.integration.http.management.IntegrationGraphController.class,
+								org.springframework.integration.handler.AbstractReplyProducingMessageHandler.RequestHandler.class
+						}),
+				@TypeHint(access = AccessBits.CLASS | AccessBits.PUBLIC_METHODS,
+						types = {
+								org.springframework.integration.xml.xpath.XPathUtils.class,
+								org.springframework.integration.json.JsonPathUtils.class,
+								com.jayway.jsonpath.JsonPath.class
+						})
+		},
+		proxies = {
+				@ProxyHint(
+						types = {
+								org.springframework.integration.gateway.RequestReplyExchanger.class,
+								org.springframework.aop.SpringProxy.class,
+								org.springframework.aop.framework.Advised.class,
+								org.springframework.core.DecoratingProxy.class
+						}),
+				@ProxyHint(
+						types = {
+								org.springframework.integration.handler.AbstractReplyProducingMessageHandler.RequestHandler.class,
```

Comment:
@aclement ,

Apparently something had changes in the project because it worked for me yesterday after unwrapping strings into their normal classes, even for this nested one.